homeassistant/components/isy994/light.py

```diff
@@ -56,7 +59,7 @@ def on_update(self, event: object) -> None:
     # pylint: disable=arguments-differ
     def turn_on(self, brightness=None, **kwargs) -> None:
         """Send the turn on command to the ISY994 light device."""
-        if brightness is None and self._last_brightness is not None:
+        if brightness is None and self._last_brightness:
```

Comment:
How is this fixing the issue that we would send the turn off command? Shouldn't the fix be that we just assume _any_ brightness and not just send `val=None`  ?

Reply:
Or can we call `self._node.on()` without passing a value ?